app/services/doc_auth/error_generator.rb

```diff
@@ -192,4 +184,12 @@ def scan_for_unknown_alerts(response_info)
       unknown_fail_count
     end
   end
+
+  def self.general_error(liveness_enabled)
```

Comment:
Yeah I agree ErrorGenerator makes more sense, especially since that's the only place this method seems to be used? Maybe even make it an instance method?

Reply:
Changed scope in `ca09545`